acceptance/tests/resource/service/ticket_4123_should_list_all_running_redhat.rb

```diff
@@ -1,12 +1,10 @@
 test_name "#4123: should list all running services on Redhat/CentOS"
+confine :to, :platform => /el-5/
```

Comment:
@johnduarte this was an issue before this change, but `el-5` only covers RedHat 5, despite the test title specifying RedHat *and* CentOS. This test is being skipped in CentOS and probably always has.